kolibri/plugins/coach/assets/src/modules/lessonsRoot/actions.js

```diff
@@ -2,7 +2,7 @@ import { LessonResource } from 'kolibri.resources';
 import router from 'kolibri.coreVue.router';
 import { createTranslator } from 'kolibri.utils.i18n';
 import useSnackbar from 'kolibri.coreVue.composables.useSnackbar';
-import { lessonSummaryLink } from '../../routes/planLessonsRouterUtils';
```

Comment:
removed the `planLessonsRouterUtils` that wasnt adding that much value, and rather making the `routes` folter a little more complex